textBasedAdventure.js

```diff
@@ -1,5 +1,439 @@
+    } else if (hearSomeVoices === "opposite") {
+        console.log("You continue to wander to find help or the finish line. ..")
+        nearingFinishline();
+    }
+}
+
+const nearingFinishline = () => {
+    console.clear();
+    let gettingClose = readline.question("Runnign along you know you're getting closer to the finish line because you see a statue that stuck out on your map. But taking a closer look oyu see someone in pain leaning up against it. Will have one last act of kindness or keep on running (stop || run)? ");
+
+        switch (gettingClose) {
+            case "run" :
+                console.log("Run, run, as fast as you can! You are almost at the end.")
+                reachedFinishline()
+                break;
+            case "stop" :
+                console.log("Woman: Thank you so much for stopping, I could really use a hand. I believe I may have pulled muscle")
+            helpRunner();
+            break;
+        }
+}
+
+const helpRunner = () => {
+    console.clear();
+    
+    let whatShouldWeDo = readline.question("Woman: I really would love it if I could walk across the line but also I do not think I can make it... Do you think you could help me hold my weight or could you just run and get help when you reach the end. Will you try to go together or will you go alone (together || alone)? ");
+
+    if (whatShouldWeDo === "alone") {
+        reachedFinishline();
+    } else if (whatShouldWeDo === "together") {
+        console.log("It's not easy but the woman is deeply greateful and apprecitive") 
+    }
+    reachedFinishline2()
+}
+
+const reachedFinishline = () => {
+    console.clear();
+    
+    console.log("You made and helped so many along the way! No matter what the final score you are awesome");
+    
+    congratulations();
+}
+const reachedFinishline2 = () => {
+    console.clear();
+    
+    console.log("Not only did you make it, but you also helped a stranded fellow racer make it as well!")
+    GSP += 50
+    
+    congratulations();
+}
+
+const congratulations = () => {
+    console.clear();
+
+    console.log(`Thanks so much for playing! Hope to see back soon! \n ${current} this is your final score!`)
+    playAgain();
+}
+
+const startOver = () => {
+    console.clear();
+
+    console.log(`Sorry ${UserInput} your run has come to an end. This is your final score ${timeCheck} \n Thanks for playing and rememebr to stop and help those around you!!!`)
+    let playAgain = readline.question("Would you like to play again (yes || no) ?")
+
+    if (playAgain === "yes") {
+        play();
+    } else {
+        console.log("Over");
+    }
+}
+
+const playAgain = () => {
+    console.clear();
+
+    let wouldYouLike = readline.question("Would you like to play again (yes || no)?")
+    if (wouldYouLike === "yes") {
+        play();
+    } else {
+        console.log("Thank you have a good day!!!")
+    }
+}
+// play()
```

Comment:
You need to have the play function running otherwise nothing on the file is going to run